### DISCUSSION TITLE
createToken is not implemented for web

### DISCUSSION THREAD

**Discussion by NeatFastro** [flutter-stripe/flutter_stripe#728] [Q&A]
Is their any particular reason why the `createToken` method is not implemented for the web platform?

**Comment by remonh87** [comment]
We didn't had yet the time to implement it yet. We recently had 2 very large syncs on the mobile sdk.  It is on the roadmap to improve the functions for the web.

**Reply by NeatFastro** [reply]
Ok, I thought it was due to some technical reasons...
I will try to implement it make a pull request, do you guys accept PRs ?

**Reply by remonh87** [reply]
yes we are always open to outside contributions 👍
